base/trace_event/trace_log.cc

```diff
@@ -2175,6 +2175,15 @@ void TraceLog::AddMetadataEventsWhileLocked() {
                                 it.second);
   }
 
+#if defined(USE_COBALT_CUSTOMIZATIONS) && !BUILDFLAG(USE_PERFETTO_CLIENT_LIBRARY)
```

Comment:
It's because we aren't actually using perfetto right now, instead we're building it so that base can build (it can't build without perfetto, but it uses a different system when `!BUILDFLAG(USE_PERFETTO_CLIENT_LIBRARY)`). Right now this is the same thing that v8 does, and we'll want to follow up to transition to using perfetto instead. I suspect this is also the cause of the v8 issue you filed a bug for